src/rp2_common/hardware_exception/include/hardware/exception.h

```diff
@@ -43,6 +43,11 @@ extern "C" {
  * ---------------------|-------|-----------------------
  * NMI_EXCEPTION        | 2     | Non Maskable Interrupt
  * HARDFAULT_EXCEPTION  | 3     | HardFault
+ * \if rp2350_specific
+ * MEMFAULT_EXCEPTION   | 4     | MemFault
```

Comment:
I think the actual exception names are:

MemManage, BusFault, UsageFault (and SecureFault (12) which we shoul add too)

![image](https://github.com/user-attachments/assets/c7bebac2-05fc-4247-bfaf-0a52f3cf6bf4)

Reply:
Fixed.

Reply:
I left out SecureFault, as that is V8 only (Table B1-4 of the Arm V7 ref manual).